src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/search.html.twig

```diff
@@ -11,7 +11,7 @@
                 <select name="method" id="method">
                     <option value="">Any</option>
                     {% for m in ['DELETE', 'GET', 'HEAD', 'PATCH', 'POST', 'PUT'] %}
-                        <option {{ m == method ? 'selected="selected"' }}>{{ m }}</option>
+                        <option {{ m in method ? 'selected="selected"' }}>{{ m }}</option>
```

Comment:
will this work fine with a string or null ?

Reply:
yes, but it does not do an equality check. It uses `str_contains`, which is not what we want

Reply:
It works in this case and is simpler than

```twig
m == method or method is iterable and m in method
```

Should I change it this way?